app/client-v2/src/stores/projectStore.ts

```diff
@@ -0,0 +1,132 @@
+import { getApiUrl } from "@/config";
+import {
+  COMPLETE_STATUS_TYPES,
+  type AnalysisStatus,
+  type ApiResponse,
+  type Project,
+  type ProjectSample,
+  type WorkerResponse,
+  WorkerResponseValueTypes
+} from "@/types/projectTypes";
+import { mande } from "mande";
+import { defineStore } from "pinia";
+import { Md5 } from "ts-md5";
+
+const baseApi = mande(getApiUrl(), { credentials: "include" });
+
+// TODO: add proper error handling. Maybe best to add error state attribute and watch accordingly cos of nested things interval/workers
```

Comment:
Got a ticket for this? Might as well add it to the epic so we we don't forget.

Reply:
good idea made ticket